src/gluonnlp/data/transforms.py

```diff
@@ -988,10 +1004,261 @@ def convert_tokens_to_ids(self, tokens):
+          sequence or the second sequence.
+        - generate valid length
+
+        For sequence pairs, the input is a tuple of 2 strings:
+        text_a, text_b.
+
+        Inputs:
+            text_a: 'is this jacksonville ?'
+            text_b: 'no it is not'
+        Tokenization:
+            text_a: 'is this jack ##son ##ville ?'
+            text_b: 'no it is not .'
+        Processed:
+            tokens:  '[CLS] is this jack ##son ##ville ? [SEP] no it is not . [SEP]'
+            type_ids: 0     0  0    0    0     0       0 0     1  1  1  1   1 1
+            valid_length: 14
+
+        For single sequences, the input is a tuple of single string: text_a.
+        Inputs:
+            text_a: 'the dog is hairy .'
+        Tokenization:
+            text_a: 'the dog is hairy .'
+        Processed:
+            text_a:  '[CLS] the dog is hairy . [SEP]'
+            type_ids: 0     0   0   0  0     0 0
+            valid_length: 7
+
+        Parameters
+        ----------
+        line: tuple of str
+            Input strings. For sequence pairs, the input is a tuple of 3 strings:
+            (text_a, text_b). For single sequences, the input is a tuple of single
+            string: (text_a,).
+
+        Returns
+        -------
+        np.array: input token ids in 'int32', shape (batch_size, seq_length)
+        np.array: valid length in 'int32', shape (batch_size,)
+        np.array: input token type ids in 'int32', shape (batch_size, seq_length)
+        """
+        # convert to unicode
+        text_a = line[0]
+        if self._pair:
+            assert len(line) == 2
+            text_b = line[1]
+
+        tokens_a = self._tokenizer(text_a)
+        tokens_b = None
+
+        if self._pair:
+            tokens_b = self._tokenizer(text_b)
+
+        if tokens_b:
+            # Modifies `tokens_a` and `tokens_b` in place so that the total
+            # length is less than the specified length.
+            # Account for [CLS], [SEP], [SEP] with "- 3"
+            _truncate_seq_pair(tokens_a, tokens_b, self._max_seq_length - 3)
+        else:
+            # Account for [CLS] and [SEP] with "- 2"
+            if len(tokens_a) > self._max_seq_length - 2:
+                tokens_a = tokens_a[0:(self._max_seq_length - 2)]
+
+        # The embedding vectors for `type=0` and `type=1` were learned during
+        # pre-training and are added to the wordpiece embedding vector
+        # (and position vector). This is not *strictly* necessary since
+        # the [SEP] token unambiguously separates the sequences, but it makes
+        # it easier for the model to learn the concept of sequences.
+
+        # For classification tasks, the first vector (corresponding to [CLS]) is
+        # used as as the "sentence vector". Note that this only makes sense because
+        # the entire model is fine-tuned.
+        tokens = []
+        segment_ids = []
+        tokens.append('[CLS]')
```

Comment:
Reminder to replace '[CLS]' with the one from BERTVocab once https://github.com/dmlc/gluon-nlp/pull/509 is merged

Reply:
Ok, I am converting a BERTVocab json file and uploading it. I will modify this once the #509 test passed and is merged.